contrib/test_jenkins.sh

```diff
@@ -466,6 +466,13 @@ run_gtest() {
 	$AFFINITY $TIMEOUT make -C test/gtest test
 	(cd test/gtest && rename .tap _gtest.tap *.tap && mv *.tap $GTEST_REPORT_DIR)
 
+    echo "==== Running malloc hooks mallopt() test ===="
+    # the test is very short. run it on every shard.
+    env UCX_IB_RCACHE=n UCX_GTEST_MALLOPT=yes \
+        GTEST_SHARD_INDEX=0 GTEST_TOTAL_SHARDS=1 \
```

Comment:
why is this needed? it's enough 1 process will do the test

Reply:
because gtest exits with non zero code if there are no tests to run

Reply:
ok